src/views/Entries/QualityControl/EntryCard/styles.scss

```diff
@@ -115,24 +116,39 @@
 
     .middle {
         flex-grow: 1;
-        padding: var(--spacing-small) var(--spacing-medium);
+        padding: var(--spacing-small) var(--spacing-large);
         overflow: auto;
 
         .excerpt-output {
             padding: 0;
             height: 240px;
         }
     }
+    .expand-button {
+        display: flex;
+        outline: none;
+    }
+    .collapse-button {
+        display: flex;
+        outline: none;
+    }
 
     .bottom {
         display: flex;
+        position: absolute;
+        bottom: calc(var(--spacing-extra-large-alt) + var(--spacing-medium-alt));
+        left: 0;
         flex-direction: column;
-        flex-shrink: 0;
-        padding: calc(var(--spacing-medium) - var(--spacing-small)) 0;
+        transition: 0.2s all ease-in-out;
+        background-color: var(--color-foreground-alt);
```

Comment:
change the background color to full white (color-foreground).
Also, add border-top for better distinction